compiler/luci/pass/src/ExpandBroadcastConstPass.test.cpp

```diff
@@ -252,3 +262,65 @@ TEST_F(ExpandBroadcastRank4ConstTest1, broadcast_impossible_NEG)
   luci::ExpandBroadcastConstPass pass;
   ASSERT_FALSE(pass.run(&_g));
 }
+
+TEST_F(ExpandBroadcastRank4ConstTest2, remove_broadcast)
```

Comment:
if possible, can we have count_of(positive) <= count_of(negative) tests?
SE requirement for unit test numbers is like this.

Reply:
I added more negative tests. It's now 6 positive test and 6 negative tests :-D